flutter/lib/src/ui/user_feedback_dialog.dart

```diff
@@ -0,0 +1,458 @@
+              ),
+              controller: nameController,
+              keyboardType: TextInputType.text,
+            ),
+            SizedBox(height: 8),
+            TextField(
+              decoration: InputDecoration(
+                border: OutlineInputBorder(),
+                hintText: 'E-mail',
+              ),
+              controller: emailController,
+              keyboardType: TextInputType.emailAddress,
+            ),
+            SizedBox(height: 8),
+            TextField(
+              minLines: 5,
+              maxLines: null,
+              decoration: InputDecoration(
+                border: OutlineInputBorder(),
+                hintText: 'What happened?',
+              ),
+              controller: commentController,
+              keyboardType: TextInputType.multiline,
+            ),
+            SizedBox(height: 8),
+            const _PoweredBySentryMessage(),
+          ],
+        ),
+      ),
+      actions: [
+        ElevatedButton(
+            onPressed: () async {
+              await (widget.hub ?? HubAdapter())
+                  .captureUserFeedback(UserFeedback(
+                eventId: widget.eventId,
+                comments: commentController.text,
+                email: emailController.text,
+                name: nameController.text,
+              ));
+              Navigator.pop(context);
+            },
+            child: Text('Submit Crash Report')),
+        TextButton(
+          onPressed: () {
+            Navigator.pop(context);
+          },
+          child: Text('Close'),
+        )
+      ],
+    );
+  }
+
+  @override
+  void dispose() {
+    nameController.dispose();
+    emailController.dispose();
+    commentController.dispose();
+    super.dispose();
+  }
+}
+
+class _PoweredBySentryMessage extends StatelessWidget {
+  const _PoweredBySentryMessage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      scrollDirection: Axis.horizontal,
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        mainAxisAlignment: MainAxisAlignment.end,
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Text('Crash reports powered by'),
+          SizedBox(width: 8),
+          SizedBox(
+            height: 30,
+            child: _SentryLogo(),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _SentryLogo extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    var color = Colors.white;
+    final brightenss = Theme.of(context).brightness;
+    if (brightenss == Brightness.light) {
+      color = Color(0xff362d59).withOpacity(1.0);
+    }
+
+    return FittedBox(
+      fit: BoxFit.contain,
+      child: CustomPaint(
+        size: Size(222, 66),
+        painter: _SentryLogoCustomPainter(color),
+      ),
+    );
+  }
+}
+
+// Created with https://fluttershapemaker.com/
+class _SentryLogoCustomPainter extends CustomPainter {
```

Comment:
This way we don't have to include it as an bitmap. Flutter doesn't has support for vector images by default.

Reply:
The image was taken from https://sentry.io/branding/